test/UniStaker.t.sol

```diff
@@ -2648,6 +2648,49 @@ contract SetAdmin is UniStakerTest {
   }
 }
 
+contract InvalidateNonce is UniStakerTest {
+  using stdStorage for StdStorage;
+
+  function testFuzz_SucessfullyInvalidatesNonceForMsgSender(address _caller, uint256 _initialNonce)
+    public
+  {
+    vm.assume(_caller != address(0));
+    vm.assume(_initialNonce != type(uint256).max);
+
+    stdstore.target(address(uniStaker)).sig("nonces(address)").with_key(_caller).checked_write(
+      _initialNonce
+    );
+
+    vm.prank(_caller);
+    uniStaker.invalidateNonce();
+
+    uint256 currentNonce = uniStaker.nonces(_caller);
+
+    assertEq(currentNonce, _initialNonce + 1, "Current nonce is incorrect");
+  }
+
+  function testFuzz_NonceIncrementsWhenCalledMultipleTimes(address _caller, uint256 _initialNonce)
```

Comment:
similarly, perhaps: "Increases the nonce by 2 when called twice"

Reply:
https://github.com/uniswapfoundation/UniStaker/pull/72/commits/335abf9be099488ad2a22ae173fe4546a0592762